src/main/java/org/opentripplanner/model/vehicle_position/RealtimeVehiclePosition.java

```diff
@@ -0,0 +1,70 @@
+package org.opentripplanner.model.vehicle_position;
+
+import java.time.Instant;
+import org.opentripplanner.model.StopLocation;
+
+/**
+ * Internal model of a realtime vehicle position.
+ */
+public class RealtimeVehiclePosition {
+
+    public final String vehicleId;
+    public final String label;
+    public final double lat;
+    public final double lon;
```

Comment:
Good point - I've changed the internal model.

In the GraphQL API stops have lat/lon rather than a coordinates field, that's why I did the same with vehicle positions. Do you agree?

Reply:
that is fine, it was just the internal model I was thinking about